<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devicePlannerForm</class>
 <widget class="QWidget" name="devicePlannerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="m_labelTitle">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select Target Device</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_labelDetail">
     <property name="text">
      <string>Select the series and device you want to target for compilation.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="15,0,5,80">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Filters</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="25,25,25">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="15,85">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="m_labelSeries">
            <property name="text">
             <string>Series:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_comboBoxSeries"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="15,85">
          <item>
           <widget class="QLabel" name="m_labelFamily">
            <property name="text">
             <string>Family:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_comboBoxFamily"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="15,85">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="m_labelPackage">
            <property name="text">
             <string>Package:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_comboBoxPackage"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxCustomLayout">
       <property name="title">
        <string>Create layout from selected device</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditPath"/>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="selectFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Create...</string>
          </property>
          <property name="icon">
           <iconset resource="../MainWindow/main_window_resource.qrc">
            <normaloff>:/images/add-circle.png</normaloff>:/images/add-circle.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Available devices:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="m_frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../MainWindow/main_window_resource.qrc"/>
  <include location="../MainWindow/main_window_resource.qrc"/>
  <include location="../MainWindow/main_window_resource.qrc"/>
 </resources>
 <connections/>
</ui>
